inventory_requests/business_logic/modify_request_cart_logic.py

```diff
@@ -10,13 +10,15 @@ def precheck_item(request, modification_type):
                 (modification_type == "approved" or modification_type == "disburse"))
 
 
-def can_approve_deny_cancel_disburse_request_cart(request_cart_to_modify, modification_type):
+def can_modify_cart_status(request):
+    return request.status == "outstanding" or (request.staff is not None and request.status == "active")
```

Comment:
added this method to determine whether the cart can be changed status from outstanding to anything else 🗡 

This is used in cancel